<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterCleanupDialog</class>
 <widget class="QDialog" name="FilterCleanupDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabWidgetPage1">
         <attribute name="title">
          <string>Remove Waypoints</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="3">
           <widget class="QToolButton" name="toolButtonRemoveNegativeSpeed">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
              <normaloff>:/icons/silk/tick.png</normaloff>:/icons/silk/tick.png</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Remove waypoint with speed higher than</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QDoubleSpinBox" name="doubleSpinBox">
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QToolButton" name="toolButtonRemoveHighSpeed">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
              <normaloff>:/icons/silk/tick.png</normaloff>:/icons/silk/tick.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Remove waypoint with Acceleration higher than</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxAcceleration">
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
            <property name="value">
             <double>9.810000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QToolButton" name="toolButtonRemoveHighAcceleration">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
              <normaloff>:/icons/silk/tick.png</normaloff>:/icons/silk/tick.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Remove waypoint which are too close (in m)</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxRadius">
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QToolButton" name="toolButtonRemoveRadius">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
              <normaloff>:/icons/silk/tick.png</normaloff>:/icons/silk/tick.png</iconset>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="4" column="0" colspan="3">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Remove waypoint which have negative speed</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Remove waypoint with altitude (in m)</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QDoubleSpinBox" name="doubleSpinBoxAltitude">
            <property name="decimals">
             <number>0</number>
            </property>
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
            <property name="value">
             <double>0.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QToolButton" name="toolButtonRemoveLowerHigherAltitude">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/silk/silk_icons.qrc">
              <normaloff>:/icons/silk/tick.png</normaloff>:/icons/silk/tick.png</iconset>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QToolButton" name="toolButtonHigherOrLowerAltitude">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string notr="true">&lt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabWidgetPage2">
         <attribute name="title">
          <string>Cleanup Waypoints</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Remove descriptions</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QToolButton" name="toolButtonRemoveDesc">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
              <normaloff>:/icons/gtk/edit-clear.png</normaloff>:/icons/gtk/edit-clear.png</iconset>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Remove every extensions</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QToolButton" name="toolButtonRemoveExtensions">
            <property name="icon">
             <iconset resource="../../GPSBook/icons/gtk/gtk_icons.qrc">
              <normaloff>:/icons/gtk/edit-clear.png</normaloff>:/icons/gtk/edit-clear.png</iconset>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="2">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../GPSBook/icons/gtk/gtk_icons.qrc"/>
  <include location="../../GPSBook/icons/silk/silk_icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FilterCleanupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FilterCleanupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
